### PR TITLE
chore: migrate to owlbot

### PR DESCRIPTION
**Pull request by Neenu1995**

### CODE REVIEW

**Review by eaball35 — APPROVED** [review]

**Comment by Neenu1995** [comment]
Merge after Kristen reviews.

File: versions.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
 google-cloud-firestore:3.0.9:3.0.10-SNAPSHOT
 google-cloud-firestore-admin:3.0.9:3.0.10-SNAPSHOT
 google-cloud-firestore-bom:3.0.9:3.0.10-SNAPSHOT
-google-cloud-firestore-parent:3.0.9:3.0.10-SNAPSHOT
```
Comment (kolea2) [inline versions.txt:7]: @Neenu1995 can you take a look why this was deleted?

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: versions.txt (hunk 1 above)
Comment (Neenu1995) [reply versions.txt:7]: the parent version is same as the google-cloud-library version. Doesn't have to be tracked on versions.txt

**Review by kolea2 — APPROVED** [review]

File: google-cloud-firestore-bom/pom.xml [hunk 2]
```diff
@@ -56,33 +56,28 @@
...
       <dependency>
         <groupId>com.google.api.grpc</groupId>
-        <artifactId>grpc-google-cloud-firestore-v1</artifactId>
-        <version>3.0.13-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-firestore-v1:current} -->
+        <artifactId>proto-google-cloud-firestore-v1</artifactId>
+        <version>3.0.13-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-firestore-v1:current} -->
       </dependency>
       <dependency>
-        <groupId>com.google.cloud</groupId>
-        <artifactId>google-cloud-firestore-admin</artifactId>
```
Comment (kolea2) [inline google-cloud-firestore-bom/pom.xml:84]: @Neenu1995 why was this dependency removed?

**Review by kolea2 — CHANGES_REQUESTED** [review]

**Review by kolea2 — APPROVED** [review]
lgtm, @eaball35 can you take a quick look?

**Comment by eaball35** [comment]
> lgtm, @eaball35 can you take a quick look?

LGTM

**Pull request merged**